### DISCUSSION TITLE
Looking to replace boiler control from Nest on Greenstar 8000 Life with Home Assistant

### DISCUSSION THREAD

**Discussion by a8ree** [emsesp/EMS-ESP32#1300] [General]
I have a Greenstar 8000 Life 40kW Combi that I currently have connected with a Nest.
Previous to installing smart TRVs, i used the controls on the Nest to turn up / down

Last year with the smart trvs installed I just wacked up the temperature on the Nest and used the Zoned_Heating integration (https://github.com/nielsfaber/zoned-heating) to switch the boiler on/off when any of the zones required heating.

The Nest isn't great in this configuration as it has a cloud API and I find that it times out fairly often - I guess when using a simple schedule it doesn't matter quite so much.

I stumbled upon this and it seems like it may be a good solution for simple local based boiler controls. Before I dive in, I thought I would see whether there were any gotchas with my boiler/intended use.

Cheers

**Comment by proddy** [comment]
It should work! What do you think @bbqkees ?

**Comment by bbqkees** [comment]
You would need to change the Nest for a EMS thermostat and then you can use
the Gateway to control it.

**Reply by a8ree** [reply]
Do I need a thermostat or can I just use Home Assistant to toggle this on/off?

**Reply by MichaelDvP** [reply]
Your zoned-heating needs a climate element to control. With a thermostat it is generated by the HA discovery, so i should work out of the box. Without thermostat you can control the boiler, but you need calculate the flowtemp and create a HA-climate control for it to make it compatible to the zoned-heating. I think switching the boiler with a fixed flowtemp is not a good solution and waste energy.

**Reply by a8ree** [reply]
I thought that the nest just flicked the switch on the boiler to call for heat or not (i.e. binary). Does it do more than that?
